Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageContentView.swift

```diff
@@ -645,6 +669,24 @@ open class ChatMessageContentView: _View, ThemeProvider {
     @objc open func handleTapOnDeliveryStatusView() {
         delegate?.messageContentViewDidTapOnDeliveryStatusIndicator(indexPath?())
     }
+    
+    // MARK: - UITextViewDelegate
+    
+    open func textView(_ textView: UITextView, shouldInteractWith URL: URL, in characterRange: NSRange, interaction: UITextItemInteraction) -> Bool {
+        if let mentionedUsers = content?.mentionedUsers, !mentionedUsers.isEmpty {
+            let tappedMentionedUser = textViewUserMentionsHandler.mentionedUserTapped(
+                on: textView,
+                in: characterRange,
+                withMentionedUsers: mentionedUsers
```

Comment:
Doesn't look to swifty

Reply:
Feel free to suggest better naming

Reply:
`with mentionedUsers: [ChatUser]` would be enough

Reply:
✅